docs/cookbooks/studio/usecases-functions.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Function use cases
+sidebar_label: Function use cases
+---
+
+The following are some of the use cases of the **Functions** section in **Studio**.
+
+## Display  weather in your bot in a caraousel node
```

Comment:
Function (code) to Fetch weather and display it in a carousel node

Reply:
Doesn't suit the steps as we are not giving the function(code) here to display the weather, we are just giving them the steps to do so. The current heading works for this.